server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/visitors/PrepareVisitor.java

```diff
@@ -0,0 +1,33 @@
+package org.opentosca.toscana.plugins.cloudfoundry.visitors;
+
+import org.opentosca.toscana.model.node.MysqlDatabase;
+import org.opentosca.toscana.model.visitor.NodeVisitor;
+
+import org.slf4j.Logger;
+
+/**
+ Prepare the model
+ */
+public class PrepareVisitor implements NodeVisitor {
+
+    public final static String CF_PREPARE_DB_PLACEHOLDER_USER = "cf_database_user_placeholder";
+    public final static String CF_PREPARE_DB_PLACEHOLDER_NAME = "cf_database_name_placeholder";
+    public final static String CF_PREPARE_DB_PLACEHOLDER_PW = "cf_database_password_placeholder";
+    //yet not sure if to set the port here
+    public final static int CF_PREPARE_DB_PLACEHOLDER_PORT = 3306;
+
+    private final Logger logger;
+
+    public PrepareVisitor(Logger logger) {
+        this.logger = logger;
+    }
+
+    @Override
+    public void visit(MysqlDatabase node) {
+        logger.debug("Prepare Model. Set placeholders for mysql database node");
+        node.setUser(CF_PREPARE_DB_PLACEHOLDER_USER);
```

Comment:
What happens in case of 2 MysqlDatabase node? I thought the placeholders need to be unique for the algorithm to work.